docker-compose.yml

```diff
@@ -331,10 +331,9 @@ services:
       - RECOMMENDATION_SERVICE_PORT
       - PRODUCT_CATALOG_SERVICE_ADDR
       - OTEL_PYTHON_LOG_CORRELATION=true
-      - OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
-      # OTEL_EXPORTER_OTLP_METRICS_ENDPOINT # Not working for Python OTLP exporter
+      - OTEL_METRICS_EXPORTER=console,otlp
       - OTEL_EXPORTER_OTLP_ENDPOINT
-      - OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE
```

Comment:
This env variable is needed in order to change the default temporality, to verify 'delta' temporality. This should be kept.

Reply:
Fixed. I removed it by accident.